paper/main.tex

```diff
@@ -703,7 +707,7 @@ \section{Illustrative Example}\label{sec:illustrative-example}
 \end{center}
 It is worth noting, however, that beyond illustrative purposes, the benefit of specifying only a maximum version for \texttt{joblib} rather than an exact version is relatively minor.
 The main advantage to relaxing a version constraint in an onion comment (when a package's behavior does not differ meaningfully between versions) is that doing so increases the likelihood that a satisfactory version will already be available in the user's Python environment, and therefore Davos will not need to install a new copy in the notebook's project directory.
-For large packages, this can be a worthwhile consideration; however \texttt{joblib} is very lightweight---less than 0.5 MB pre-built, with no required dependencies.
+For large packages, this can be a worthwhile consideration; however \texttt{joblib} is very lightweight---less than 0.5 MB pre-built, with no other dependencies.
```

Comment:
"required dependencies" was an intentional phrasing here because se draw on the fact that `joblib` has an **optional** dependency on `numpy` 2 paragraphs below this. This is also how `joblib` itself makes the distinction: https://pypi.org/project/joblib/

Reply:
got it.  ok, feel free to revert